src/navigation/NavBar.js

```diff
@@ -26,77 +25,73 @@ const NavLink = ({
   const href = `#${to}`
   const active = exact ? hash === href : hash && hash.startsWith(href)
   const anchorClass = classnames({
-    'bg-white-10': active,
+    'bg-white-10 navbar-item-active': active,
     'o-50 no-pointer-events': disabled
-  }, ['dib db-l pv3 white no-underline f5 hover-bg-white-10 tc'])
+  }, ['dib db-l pt2 pb3 pv3-l white no-underline f5 hover-bg-white-10 tc bb bw2 bw0-l b--navy'])
   const svgClass = classnames({
     'o-100': active,
     'o-50': !active
   }, ['fill-current-color'])
 
   return (
     // eslint-disable-next-line jsx-a11y/anchor-is-valid
-    <a href={disabled ? null : href} className={anchorClass} style={{ borderLeft: active ? '5px solid rgba(201, 210, 215, .4)' : '' }} role='menuitem' title={children}>
-      <span className={`dib ${open ? 'dt-l' : ''}`}>
-        <span className={`dib dtc-l v-mid ${open ? 'pl3 pl5-l' : 'ph3'}`} style={{ width: 50 }}>
+    <a href={disabled ? null : href} className={anchorClass} role='menuitem' title={children}>
+      <div className='db ph2 pv1'>
+        <div className='db'>
           <Svg width='50' className={svgClass} />
-        </span>
-        <span className={`${open ? 'dib dtc-l' : 'dn'} pl2 pl3-l pr3 tl-l v-mid `}>
+        </div>
+        <div className={`${active ? 'o-100' : 'o-50'} db f6 tc montserrat ttu fw1 `}>
```

Comment:
I would add `style={{ whiteSpace: 'pre-wrap' }}` here to do this:
![image](https://user-images.githubusercontent.com/24696635/88200263-d5c8cf00-cc3d-11ea-8dcc-f0951c68fa11.png)

Instead of the actual:
![image](https://user-images.githubusercontent.com/24696635/88200292-de210a00-cc3d-11ea-9e35-31aaaea1fd48.png)


But up to you 😄

Reply:
Awesome 😊  thank you!